kalite/i18n/management/commands/create_dummy_language_pack.py

```diff
@@ -0,0 +1,100 @@
+"""
+
+The create_dummy_language_pack command downloads the 'en' language
+pack from the central server and creates a new language pack based on
+that. Make sure you have internet!
+
+"""
+
+import accenting
+import json
+import os
+import polib
+import requests
+import tempfile
+import zipfile
+from cStringIO import StringIO
+
+from django.conf import settings
+from django.core.management.base import NoArgsCommand
+
+from fle_utils.general import ensure_dir
+from kalite.i18n import get_language_pack_url, get_locale_path
+from kalite.version import VERSION
+
+logging = settings.LOG
+
+BASE_LANGUAGE_PACK = "en"       # language where we base the dummy langpack from
+TARGET_LANGUAGE_PACK = "eo"     # what the "dummy" language's language code. Will be. Sorry, Esperantists.
+TARGET_LANGUAGE_DIR = get_locale_path(TARGET_LANGUAGE_PACK)
+MO_FILE_LOCATION = os.path.join(TARGET_LANGUAGE_DIR, "LC_MESSAGES")
+TARGET_LANGUAGE_METADATA_PATH = os.path.join(
+    TARGET_LANGUAGE_DIR,
+    "%s_metadata.json" % TARGET_LANGUAGE_PACK,
+)
+
+
+class Command(NoArgsCommand):
+
+    def handle_noargs(self, **options):
+        logging.info("Creating a debugging language pack, with %s language code." % TARGET_LANGUAGE_PACK)
+        langpack_zip = download_language_pack(BASE_LANGUAGE_PACK)
+        django_mo_contents, djangojs_mo_contents = retrieve_mo_files(langpack_zip)
+        dummy_django_mo, dummy_djangojs_mo = (create_mofile_with_dummy_strings(django_mo_contents, filename="django.mo"),
+                                              create_mofile_with_dummy_strings(djangojs_mo_contents, filename="djangojs.mo"))
+        logging.info("Finished creating debugging language pack %s." % TARGET_LANGUAGE_PACK)
+
+
+def download_language_pack(lang):
+    logging.debug("Downloading base language pack %s for creating the debugging language." % BASE_LANGUAGE_PACK)
+    url = get_language_pack_url(lang)
+
+    try:
+        resp = requests.get(url)
+        resp.raise_for_status()
+    except requests.ConnectionError as e:
+        logging.error("Error downloading %s language pack: %s" % (lang, e))
+
+    logging.debug("Successfully downloaded base language pack %s" % lang)
+    return zipfile.ZipFile(StringIO(resp.content))
```

Comment:
So if an exception is caught you'll execute both these lines (58 & 59) anyway? What is the value of `resp.content` if a ConnectionError is raised? (Namely, will it cause an error in line 59?)

Perhaps you could move lines 58 & 59 into your try block and return something else in the case of an exception. (Maybe `None` or an empty ZipFile?)